core/domain/email_manager.py

```diff
@@ -351,6 +351,117 @@
         'Review')
 }
 
+NOTIFICATION_FOR_CONTRIBUTOR_RANKING_ACHIEVEMENT: Dict[str, Dict[str, str]] = {
+    feconf.CONTRIBUTION_TYPE_TRANSLATION: {
+        feconf.CONTRIBUTION_SUB_TYPE_ACCEPTANCE: {
```

Comment:
"subtype" is one word, please fix throughout.

Reply:
Done